src/composables/useTransactions.ts

```diff
@@ -22,7 +22,7 @@ import { GnosisTransactionDetails } from './trade/useGnosis';
 
 const WEEK_MS = 86_400_000 * 7;
 // Please update the schema version when making changes to the transaction structure.
-const TRANSACTIONS_SCHEMA_VERSION = '1.1.1';
+const TRANSACTIONS_SCHEMA_VERSION = '1.1.2';
```

Comment:
I have changed the schema for approval transaction - so I'm bumping it. (`tokenAddress` -> `contractAddress`)